<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="settingsTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="GlobalSettingsWidget" name="globalSettings"/>
      <widget class="ScraperSettingsWidget" name="scraperSettings"/>
      <widget class="MovieSettingsWidget" name="movieSettings"/>
      <widget class="TvShowSettingsWidget" name="tvShowSettings"/>
      <widget class="ConcertSettingsWidget" name="concertSettings"/>
      <widget class="MusicSettingsWidget" name="musicSettings"/>
      <widget class="KodiSettingsWidget" name="kodiSettings"/>
      <widget class="NetworkSettingsWidget" name="networkSettings"/>
      <widget class="ExportSettingsWidget" name="exportSettings"/>
      <widget class="ImportSettingsWidget" name="importSettings"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
      <item>
       <spacer name="horizontalSpacerButtons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGlobal"/>
   <addaction name="actionScraper"/>
   <addaction name="actionMovies"/>
   <addaction name="actionTV_Shows"/>
   <addaction name="actionConcerts"/>
   <addaction name="actionMusic"/>
   <addaction name="actionXBMC"/>
   <addaction name="actionNetwork"/>
   <addaction name="actionExport"/>
   <addaction name="actionImport"/>
  </widget>
  <action name="actionMovies">
   <property name="text">
    <string>Movies</string>
   </property>
   <property name="page" stdset="0">
    <number>2</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">film</string>
   </property>
  </action>
  <action name="actionTV_Shows">
   <property name="text">
    <string>TV Shows</string>
   </property>
   <property name="page" stdset="0">
    <number>3</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">monitor</string>
   </property>
  </action>
  <action name="actionConcerts">
   <property name="text">
    <string>Concerts</string>
   </property>
   <property name="page" stdset="0">
    <number>4</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">micro</string>
   </property>
  </action>
  <action name="actionGlobal">
   <property name="text">
    <string>Global</string>
   </property>
   <property name="page" stdset="0">
    <number>0</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">tools</string>
   </property>
  </action>
  <action name="actionXBMC">
   <property name="text">
    <string>Kodi</string>
   </property>
   <property name="page" stdset="0">
    <number>6</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">network</string>
   </property>
  </action>
  <action name="actionNetwork">
   <property name="text">
    <string>Network</string>
   </property>
   <property name="page" stdset="0">
    <number>7</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">world</string>
   </property>
  </action>
  <action name="actionScraper">
   <property name="text">
    <string>Scraper</string>
   </property>
   <property name="page" stdset="0">
    <number>1</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">search</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="page" stdset="0">
    <number>8</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">cloud_upload</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import</string>
   </property>
   <property name="page" stdset="0">
    <number>9</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">download</string>
   </property>
  </action>
  <action name="actionMusic">
   <property name="text">
    <string>Music</string>
   </property>
   <property name="toolTip">
    <string>Music</string>
   </property>
   <property name="page" stdset="0">
    <number>5</number>
   </property>
   <property name="iconName" stdset="0">
    <string notr="true">music</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GlobalSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/GlobalSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExportSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/ExportSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImportSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/ImportSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScraperSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/ScraperSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MovieSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/MovieSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/MusicSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KodiSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/KodiSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConcertSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/ConcertSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TvShowSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/TvShowSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NetworkSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/NetworkSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionGlobal</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConcerts</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMovies</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNetwork</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScraper</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTV_Shows</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionXBMC</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExport</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMusic</sender>
   <signal>triggered()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>onAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAction()</slot>
 </slots>
</ui>
